packages/shared/src/api/QuestionGenerator.ts

```diff
@@ -51,6 +51,18 @@ export interface MultipleChoiceQuestion extends QuestionBase {
 
   /** The feedback function for this question; defaults to undefined */
   feedback?: MultipleChoiceFeedbackFunction
+
+  /** Whether the question is a matching question; defaults to false */
+  matching?: boolean
```

Comment:
This seems to change the MultipleChoiceQuestion into a new question type. This should be just a new question type